pkg/events/grpc/grpc.go

```diff
@@ -141,7 +141,7 @@ func (srv *EventsServer) Stream(req *ttnpb.StreamEventsRequest, stream ttnpb.Eve
 	store, hasStore := srv.pubsub.(events.Store)
 	var group *errgroup.Group
 	if hasStore {
-		if req.After == nil {
+		if req.After == nil && req.Tail == 0 {
```

Comment:
I initially wanted to come back to the old diff, where `srv.pubsub.Subscribe` would have been used if both of these are `nil`/`0`, but I think that there was some thought behind the original change. 

Is listening via the streams cheaper / more optimal than using the PubSub ?

Reply:
My main reasoning was that I wanted to follow the same subscription method for all subscriptions in a deployment.

Secondary reason was that with the streams (instead of the pub/sub) we can do some rate limiting on each stream. We wouldn't be able to do that with the other subscription method because it relies on a single `redis.(*PubSub).subscribeTask` routine.